src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/UuidToStringTransformerTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Extension\Core\DataTransformer;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\Extension\Core\DataTransformer\UuidToStringTransformer;
+use Symfony\Component\Uid\Uuid;
+
+class UuidToStringTransformerTest extends TestCase
+{
+    public function dataProvider()
```

Comment:
better rename this to `provideValidUuid` (so we can also have a data provider for invalid UUIDs)

Reply:
fixed